Documentation/ApiOverview/UpdateWizards/Creation.rst

```diff
@@ -6,21 +6,23 @@
 .. _upgrade-wizard-interface:
 
 ================================
-Creating Generic Upgrade Wizards
+Creating generic upgrade wizards
```

Comment:
@marble These are great changes, but some of them change a bit more than the original PR. The original PR was for structural changes. Making off-topic changes makes it more difficult to resolve merge conflicts and review the PR. 

Fixed the merge conflicts.  (The merge conflicts are my bad because I made several separate PR for the same page. Perhaps next time it would be better to wait for the merge. But in general it does help to keep the PR "clean" and not add unrelated changes).